File: programacao/c/respostas/operacoes/41.c
#include <stdio.h>

int main() {
    int n, j, m, i, resultado;

    printf("Digite o valor de n: ");
    scanf("%d", &n);

    printf("Digite o valor de j: ");
    scanf("%d", &j);

    printf("Digite o valor de m: ");
    scanf("%d", &m);

    printf("Os %d primeiros naturais congruentes a %d modulo %d:\n", n, j, m);
    for (i = 0; i < n; i++) {
        resultado = j + i * m;
        printf("%d ", resultado);
    }
    printf("\n");

    return 0;
}

File: programacao/python/respostas/operacoes/41.py
n = int(input("Valor de n: "))
j = int(input("Valor de j: "))
m = int(input("Valor de m: "))

print(f"Os {n} primeiros naturais congruentes a {j} modulo {m}:")
for i in range(n):
    print(j + i * m, end=" ")
print()
